### ISSUE TITLE
Question about a value in MobileNeRF

### ISSUE THREAD

**Issue by G-1nOnly** [google-research/jax3d#122]
Hello, thanks for contributing such a great work.
I'm wondering is it possible for us to obtain the "T" values in the **compute_undc_intersection** in the code, here the "T" values correspond with the value that we express **"_points = origin + direction * T_"** in the normal NeRF ray-sampling process. If it's possible, could you leave some idea here? 
Thanks again

**Comment by flyingshan** [comment]
I think the **world_tx** in compute_undc_intersection denotes the T.

**Comment by G-1nOnly** [comment]
@flyingshan  I think it somehow mismatches the "T" for the original value which lies in [near, far] but the value of world_tx could be very large sometimes, is it because of the sampling way is different?

**Comment by flyingshan** [comment]
I think you are right. I check the code again. The **world_tx** just indicates how far the intersection points are away from the origin. I  think you can calculate the real T value from the **ray_origin, ray_directions, and the world_positions** through **points = origin + direction * T**.

**Comment by G-1nOnly** [comment]
Yeah, I've tried this way, but the "T" caculated seems not correspond to all the points by this way, I think it's because the uniqueness of the sampleing method since normally we get T, origin and direction to calculate points but MobileNeRF gets the points in its own way and when calculating the weights, MobileNeRF directly optimizes "alpha" instead of sigmas normally in NeRF.
